pages/common/lychee.md

```diff
@@ -0,0 +1,24 @@
+# lychee
+
+> Find broken URLs.
+> More information: <https://lychee.cli.rs/usage/cli/>.
+
+- Scan a website for broken links:
+
+`lychee {{https://example.com}}`
+
+- Display breakdown of error types:
+
+`lychee --form detailed {{https://example.com}}`
```

Comment:
`--form` is not an option.

> --format <FORMAT>
          Output format of final status report (compact, detailed, json, markdown) [default: compact]


```suggestion
- Display a breakdown of error types:

`lychee --format detailed {{https://example.com}}`
```